src/pages/workspaces/workspace/useDeleteWorkspaceState.ts

```diff
@@ -43,7 +43,7 @@ export interface DeleteWorkspaceHookArgs {
 
 export const useDeleteWorkspaceState = (hookArgs: DeleteWorkspaceHookArgs): DeleteWorkspaceState => {
   const [deleting, setDeleting] = useState(false);
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
This is to avoid the initial flash of the dialog before we enter the loading state. I wonder why this wasn't set to `true` initially… I'm a bit nervous that there's some case where having an initial value of `false` will be an issue.